code/game/objects/items/weapons/storage/bible.dm

```diff
@@ -213,3 +246,98 @@
+	cultist = deconvertee.mind.GetRole(CULTIST)
+	cultist.deconversion = src
+
+	deconvertee.eye_blurry = max(deconvertee.eye_blurry, 10)
+	deconvertee.Dizzy(30)
+	deconvertee.stuttering = max(deconvertee.stuttering, 10)
+	deconvertee.Jitter(30)
+	deconvertee.Knockdown(10)
+
+	if (istype(bible.my_rel, /datum/religion/cult))
+		lol = TRUE
+	var/datum/faction/bloodcult/cult = find_active_faction_by_type(/datum/faction/bloodcult)
+	var/living_cultists = 0
+	for(var/datum/role/cultist/C in cult.members)
+		if (C.antag && C.antag.current && C.antag.current.stat != DEAD)
+			living_cultists++
+	if (living_cultists <= 1)
+		last_cultist = TRUE
+
+	spawn()
+		spawn()
+			if (alert(deconvertee, "You are being compelled by the powers of [bible.my_rel.deity_name][lol ? " (wait what?)" : ""] to give up on serving the Cult of Nar-Sie[lol ? " (huh!?)" : ""]","You have 10 seconds to decide","[!lol ? "Abandon the Cult" : "I am so confused right now, ok I guess?"]","[!lol ? "Resist!" : "This is obviously a trick! Resist!"]") == "[!lol ? "Abandon the Cult" : "I am so confused right now, ok I guess?"]")
+				success = DECONVERSION_ACCEPT
+				if (!target && !last_cultist)//no threats if nobody remains to carry them out.
+					to_chat(deconvertee, "<span class='sinister'>[lol ? "WERE YOU DECEIVED THAT EASILY? SO BE IT THEN." : "THERE WILL BE A PRICE."]</span>")
+			else
+				success = DECONVERSION_REFUSE
+				if (!target)
+					to_chat(deconvertee, "<span class='warning'>You block the sweet promises of forgiveness from your mind.</span>")
+		sleep(100)
+		if (!deconvertee || !iscultist(deconvertee))
+			qdel(src)
+			return
+		deconvertee.take_overall_damage(10)//it's a painful process no matter what.
+		var/turf/T = get_turf(deconvertee)
+		anim(target = deconvertee, a_icon = 'icons/effects/effects.dmi', flick_anim = "cult_jaunt_land", lay = SNOW_OVERLAY_LAYER, plane = EFFECTS_PLANE)
+		var/mob/living/simple_animal/hostile/shade/redshade_A = new(T)
+		var/mob/living/simple_animal/hostile/shade/redshade_B = new(T)
+		if (!bible.my_rel.leadsThisReligion(deconverter))//the shades are a bit stronger if it's not an actual chaplain doing the deconversion, or they're not using a bible of their religion.
+			redshade_A.buff()
+			redshade_B.buff()
+		var/list/adjacent_turfs = list()
+		for (var/turf/U in orange(1,T))
+			adjacent_turfs += U
+		switch(success)
+			if (DECONVERSION_ACCEPT)
+				playsound(deconvertee, 'sound/effects/deconversion_complete.ogg', 50, 0, -4)
+				to_chat(deconvertee,)
+				deconvertee.visible_message("<span class='notice'>You see [deconvertee]'s eyes become clear. Through the blessing of [lol ? "some fanfic headcanon version of [bible.my_rel.deity_name]" : "[bible.my_rel.deity_name]"] they have renounced Nar-Sie.</span>","<span class='notice'>You were forgiven by [bible.my_rel.deity_name]</span><span class='sinister'>[lol ? " (YEAH RIGHT...)" : ""]</span><span class='notice'>. You no longer share the cult's goals.</span>")
```

Comment:
I guess we're going with "fanfic headcanon" in actual messages that get displayed in game huh

Reply:
the existence of Nanotrasen-endorsed "chaplains of Nar-Sie" is comedic enough to warrant a few 4th wall-leaning easter eggs.